packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -177,6 +177,10 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
       setActivePane('none');
     } else {
       setActivePane('chat');
+      // timeout is required to give the window time to render the sendbox so we have something to send focus to.
+      setTimeout(() => {
+        (document.querySelector(`[data-ui-id="sendbox-textfield"]`) as HTMLTextAreaElement)?.focus();
+      }, 10);
```

Comment:
How did we decide that 10 ms is enough to render sendbox?

Reply:
We shouldn't add non-deterministic behaviour to production code.

To me it seems like the `'chat'` pane should auto-focus when it is shown. Similar to how `SendBox` does it (b.t.w, doesn't this conflict with the auto-focus on `SendBox`?

Reply:
I can make it longer but the 10ms works out because its CSS changing. if I make it longer it dosen't focus right away and if I make it shorter it wont focus at all so 10ms worked out for it to appear natural.